# This just adds a message reminding users that this will be
# removed.
# This will not be sent upstream.
--- perl-5.16.3/pod/perl.pod.orig       Thu Jun 11 10:28:40 2015
+++ perl-5.16.3/pod/perl.pod    Thu Jun 11 10:29:52 2015
@@ -479,3 +479,7 @@
 The three principal virtues of a programmer are Laziness,
 Impatience, and Hubris.  See the Camel Book for why.
 
+=head1 CAVEAT
+
+Perl 5.16 is now Obsolete, and will be removed from a future
+release of Solaris.  Please migrate any consumers to use Perl 5.20.
